roles/ee_builder/tasks/00_build_ee.yml

```
@@ -70,7 +70,7 @@
     tag: "{{ __execution_environment_definition.tag | default(omit) }}"
     validate_certs: "{{ ee_validate_certs | default(omit) }}"
     push_args:
-      dest: "{{ ee_registry_dest }}"
+      dest: "{{ ee_registry_dest }}/{{ __execution_environment_definition.name }}{{ ':' + __execution_environment_definition.tag if __execution_environment_definition.tag else '' }}"
```

I had a play around and this code currently fails if there is no tag defined. The best way I found to resolve this is to just default `.tag` to `latest` (which is what the module does) 
```suggestion
      dest: "{{ ee_registry_dest }}/{{ __execution_environment_definition.name }}:{{ __execution_environment_definition.tag | default('latest') }}"
```

---

Hi, thanks for the review! The order of precedence for inline if-else does work, but as you discovered I didn't actually check for the tag being undefined. I also thought about defaulting to latest as it's more clear. I pushed an alternative using 'is defined', but happy to change it to default to latest.

---

No, I'm happy with what you have added here. It's more flexible to changes in the upstream (I'm sure the default won't change from `latest`, but it could. 

Thanks for your contribution